frontend/annotation_resolver.py

```diff
@@ -65,27 +69,27 @@ def resolve(self, annotation):
         if annotation[:4] == "List":
```

Comment:
Yes you are right, using the AST right away was an easier solution. I think I was acting stupid 😅 Should I change it back to use AST nodes? Provided that it would be almost the same code size.